mlflow/azure/client.py

```diff
@@ -26,7 +26,7 @@ def put_adls_file_creation(sas_url, headers):
     """
     request_url = _append_query_parameters(sas_url, {"resource": "file"})
 
-    request_headers = {}
+    request_headers = {"Content-Length": "0"}
```

Comment:
@smurching is this change that should be put into production or is it just a debugging thing?

Reply:
I think I needed this change for the presigned URL to work, you could try without it